### DISCUSSION TITLE
CRITICAL: Could not find QtWebEngineProcess

### DISCUSSION THREAD

**Discussion by jamfor999** [qutebrowser/qutebrowser#5750] [Q&A]
Hi,
I'm trying to run Qutebrowser on Arch Linux. It appears like PyQT 5.15 has broken compatibility?
This is the issue I get when either compiling qutebrowser manually, compiling it using an AUR helper, downloading the release and installing it myself, or using the official qutebrowser package on Arch:

`[jf352@jf352desktop ~]$ qutebrowser
16:09:13 INFO: Run :adblock-update to get adblock lists.
16:09:13 CRITICAL: Could not find QtWebEngineProcess
Aborted (core dumped)
`

I notice this is very similar to https://github.com/qutebrowser/qutebrowser/issues/5273

`qutebrowser --backend webkit` works fine, with qt5-webkit installed, but WebEngine is the default.

**Comment by The-Compiler** [comment]
This is an issue with your system most likely, not Arch Linux/qutebrowser/(Py)Qt 5.15 - thus I have converted it to a discussion.

Can you please show `which python3` as well as:

```
python3 -c "from PyQt5 import QtCore; print(QtCore.__file__); print(QtCore.qVersion()); print(QtCore.QLibraryInfo.location(QtCore.QLibraryInfo.DataPath))"
```

**Reply by jamfor999** [reply]
Sure -

[jf352@jf352desktop ~]$ which python3
/usr/bin/python3
[jf352@jf352desktop ~]$ python3 -c "from PyQt5 import QtCore; print(QtCore.__file__); print(QtCore.qVersion()); print(QtCore.QLibraryInfo.location(QtCore.QLibraryInfo.DataPath))"
/home/jf352/.local/lib/python3.8/site-packages/PyQt5/QtCore.abi3.so
5.15.1
/home/jf352/.local/lib/python3.8/site-packages/PyQt5/Qt
[jf352@jf352desktop ~]$

**Reply by The-Compiler** [reply]
It looks like you did `pip install PyQt5` so qutebrowser is picking up Qt from `/home/jf352/.local/lib/python3.8/site-packages/PyQt5/Qt` instead of the system-wide Archlinux one. Unless there's a good reason for this, you might want to `pip uninstall PyQt5`.
